content/blog/2022-08-31-announcing-poetry-1-2-0.md

```diff
@@ -0,0 +1,793 @@
+- Projects have long had the time to migrate to Python 3.
+
+{{% note %}}
+If you rely on Poetry for a Python 2.7 project, the Poetry 1.1 branch is still available, though it will no longer be
+maintained.
+{{% /note %}}
+
+### Dropping support for Python 2.7, 3.5 and 3.6 as runtimes
+
+Poetry 1.2 drops runtime support for Python 2.7, 3.5 and 3.6. Running Poetry on these versions is now untested and
+unsupported.
+
+{{% note %}}
+This change is about installing and running Poetry itself. Managing projects requiring Python 3.5 and 3.6, as well as
+older Python 3 versions is still supported.
+{{% /note %}}
+
+### Dependency groups
+
+Poetry provides a way to **organize** your dependencies by **groups**. For instance, you might have dependencies that
+are only needed to test your project or to build the documentation.
+
+To declare a new dependency group, use a `tool.poetry.group.<group>` section where `<group>` is the name of your
+dependency group (for instance, `test`):
+
+```toml
+[tool.poetry.group.test]  # This section can be omitted
+
+[tool.poetry.group.test.dependencies]
+pytest = "^7.1.0"
+pytest-mock = "*"
+```
+
+{{% note %}}
+All dependencies **must be compatible across all groups** as they will be resolved regardless of whether they are
+selected for installation or not (see [Installing group dependencies](#installing-group-dependencies)).
+
+Think of dependency groups as **labels** associated with your dependencies: as all groups will be installed **by
+default**, they are simply a way to organize the dependencies logically.
+{{% /note %}}
+
+The dependencies declared in `tool.poetry.dependencies` are part of an implicit `main` group.
+
+```toml
+[tool.poetry.dependencies]  # The implicit `main` dependency group
+httpx = "*"
+pendulum = "*"
+
+[tool.poetry.group.test.dependencies]
+pytest = "^7.1.0"
+pytest-mock = "*"
+```
+
+Dependency groups, other than the implicit `main` group, should generally contain additional dependencies that are part
+of your development process, as installing them is only possible using Poetry and `poetry install`.
+
+If your project has additional dependencies meant to add additional functionality at runtime, they should be declared
+using the ecosystem-standard [extras][extras docs] instead. Extras are supported by
+package build and install tools such as `pip`.
+
+[extras docs]: {{< relref "docs/1.2/pyproject#extras" >}}
+
+{{% note %}}
+**A note about the `dev-dependencies` section**
+
+Any dependency declared in the legacy `dev-dependencies` section will automatically be added to a `dev` group.
+Thus, the following examples are equivalent:
+
+```toml
+[tool.poetry.dev-dependencies]
+pytest = "^7.1.0"
+pytest-mock = "*"
+```
+
+```toml
+[tool.poetry.group.dev.dependencies]
+pytest = "^7.1.0"
+pytest-mock = "*"
+```
+
+As the `dev-dependencies` is now deprecated, projects should migrate to the new `group` syntax as soon as possible.
```

Comment:
Maybe we could mention that 1.1 will not be able to understand this new syntax?

Reply:
It's a little redundant, but can't hurt.